openmsupply/mobile
# #4203 Set vaccinator as prescriber for dispensing history modal if available

Fixes #4203

## Change summary

- Small change to set the vaccinator as the prescriber for the Dispensing History modal if available when fetching remote history records (since this modal only displays a prescriber column)

![image](https://user-images.githubusercontent.com/65875762/125370962-d4a32b80-e3d3-11eb-8cb4-7ed5349c1d35.png)

## Testing

Steps to reproduce or otherwise test the changes of this PR:

- Prerequisites:
  - Run `z_toggleDangerousPatientMode` (⚠️!)  on mSupply Desktop
  - Create a vaccination on a store (ensure it has a vaccinator selected)
  - Login to a different store on mobile and navigate to Dispensing -> History

- [ ] Confirm that the vaccine history record shows up, and the vaccinator is displayed

### Related areas to think about
Not sure if it is super clear that a prescriber/vaccinator are interchangeable on this page.
Some possible ways to make it clearer (but weren't done as I wasn't sure...):
- Adding an extra 'vaccinator' column (but probably too much clutter).
- Changing the column heading to 'Prescriber/Vaccinator' if vaccine module is enabled?
